acctestutil/vaulthelper.go

```diff
@@ -0,0 +1,76 @@
+package acctestutil
+
+import (
+	"testing"
+
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+)
+
+type CompareVaultVersionFunc func(*version.Version) bool
+
+// SkipIfVaultVersionLT skips of the running vault version is less-than ver.
+func SkipIfVaultVersionLT(t *testing.T, ver *version.Version) {
```

Comment:
Thanks for working on these! I tested these with the legacy resources and these can indeed work with the legacy resources as well

should we have the names match with the deprecated methods? I was testing these with the legacy resources and thought that it might be an easier refactor down the line when we choose to move them over if the names were 1-1 😅 

example `SkipIfVaultVersionLT -> SkipIfAPIVersionLT`